example/client.php

```diff
@@ -29,18 +32,15 @@
 //
 $worker = new Worker($transport, $loop);
 
+$worker->setLogger($logger);
```

Comment:
we will need per workflow logger as well, with replaying capabilities

Reply:
It should be located somewhere here: https://github.com/temporalio/php-sdk/blob/develop/src/Runtime/Workflow/Executor.php#L79